content/legal/index.md

```diff
@@ -5,6 +5,47 @@ title: Legal
 constellation: constellation-01.svg
 ---
 
-Report Copyright Infringement or Abuse on the IPFS Gateway Service: [abuse@ipfs.io](mailto:abuse@ipfs.io)
+### Report Copyright Infringement on the IPFS Gateway Service
+Protocol Labs Inc. and the core IPFS team have adopted this copyright infringement policy in accordance with the [Digital Millennium Copyright Act](http://http://lcweb.loc.gov/copyright/legislation/dmca.pdf).
 
-For all other inquiries, contact: [legal@ipn.io](mailto:legal@ipn.io)
+If you are a copyright owner, or agent of the owner, and you believe that content available on the IPFS Gateway Service infringes one or more of your copyrights please notify us by sending a DMCA Notice of the infringement containing the information below.    After we receive a valid and complete notice, we will investigate and block or otherwise make the material unavailable through the IPFS Gateway Service. Learn more about our process below.
+
+The email address (preferred) and physical address (less preferred) of our Designated Agent to receive notification of claimed infringement is listed at the end of this policy.  Please read it carefully and make sure you are aware of the legal implications of the DMCA. You can be held liable for damages, including costs and attorney fees, if you materially misrepresent that material or activity infringes on your copyright.  We may highlight such abuses, too. Consult an attorney before filing a DMCA Notice if you are unsure whether your copyrights have been infringed, or if you unsure whether your materials were removed in error.
```

Comment:
```suggestion
The email address (preferred) and physical address (less preferred) of our Designated Agent to receive notification of claimed infringement is listed at the end of this policy.  Please read it carefully and make sure you are aware of the legal implications of the DMCA. You can be held liable for damages, including costs and attorney fees, if you materially misrepresent that material or activity infringes on your copyright.  We may highlight such abuses, too. Consult an attorney before filing a DMCA Notice if you are unsure whether copyrights have been infringed, or if you are unsure whether your materials has been removed in error.
```

(should probably also be "whether or not" but just "whether" may be "technically correct").

Reply:
I can't figure out how to make a suggested change to a suggested change but it should be ... "your materials have been removed" yeah?

Reply:
Ah, yes...